### PR TITLE
SEAR-169_SEAR-272_Compliance_History_Mapper_Adj: Changed the Allegati…

### PR DESCRIPTION
**Pull request by pradipforever**
Changed the Allegation Mapper and added dictionary files to change the current Json format

### JIRA Issue Link
- https://osi-cwds.atlassian.net/browse/CALS-NNNN

### Technical Description
Changed Allegation DTO class to output the correct Json format
Added two ENUM dictionary classes to output code and description for allegation resolution type and complaint type
Added logic in Complaint Mapper to map the code and description for both complaint and allegation

### Tests
- [X] I have included unit tests 
- [ ] I have included integration tests 
- [ ] I have included performance tests 
- [ ] I have included other tests 
- [ ] I have NOT included tests 

### Types of changes

- [X] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Technical debt
Need to change code and description from the database instead of reading from ENUM dictionary classes to output code and description for allegation resolution type and complaint type

### CODE REVIEW

**Comment by pradipforever** [comment]
retest this please

File: src/main/java/gov/ca/cwds/cals/service/mapper/ComplaintMapper.java [hunk 1]
```diff
@@ -16,12 +20,11 @@
 /**
  * @author CWDS CALS API Team
  */
-
 @Mapper(uses = TrailingSpacesRemovalPostMappingProcessor.class)
 public interface ComplaintMapper {
     ComplaintMapper INSTANCE = Mappers.getMapper(ComplaintMapper.class);
 
-    @Mapping(target = "messages", ignore = true)
+    @Mapping(target = "messages")
```
Comment (AlexanderSerbin) [inline src/main/java/gov/ca/cwds/cals/service/mapper/ComplaintMapper.java]: I meant removing all the line not just ignore = true part of line

File: src/main/java/gov/ca/cwds/cals/service/mapper/ComplaintMapper.java [hunk 2]
```diff
@@ -35,106 +38,193 @@
     @Mapping(source = "crp2Followup", target = "followupComments")
     ComplaintDTO entityToDTO(ComplaintReportLic802 complaint);
 
-    @Mapping(target = "messages", ignore = true)
+    @Mapping(target = "messages")
```
Comment (AlexanderSerbin) [inline src/main/java/gov/ca/cwds/cals/service/mapper/ComplaintMapper.java]: I meant removing all the line not just ignore = true part of line

**Comment by pradipforever** [comment]
retest this please

**Pull request merged**